src/foraging_gui/Foraging.py

```diff
@@ -223,9 +222,6 @@ def __init__(self, parent=None,box_number=1,start_bonsai_ide=True):
         # Initializes session log handler as None
         self.session_log_handler = None
 
-        # generate an upload manifest when a session has been produced
-        self.upload_manifest_slot = self.sessionGenerated.connect(self._generate_upload_manifest)
```

Comment:
Why remove the signal/slot architecture Micah added?

Reply:
I guess that structure is used for running the "_generate_upload_manifest" only once, which could be problematic (e.g. when we start the video recording after the first trial, the video modality will not be recorded.)